pkgs/by-name/js/json2kdl/package.nix

```diff
@@ -0,0 +1,28 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+}:
+
+let
```

Comment:
no need for `let` here, `rec` is more readable and consistent with the rest of nixpkgs